config/migrations/2024/bestuurseenheden/20240919101332-add-new-welzijnsvereniging-sint-anna/20240919101332-add-new-organisation-sint-anna.sparql

```diff
@@ -0,0 +1,54 @@
+PREFIX rdf:   <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+PREFIX skos:  <http://www.w3.org/2004/02/skos/core#>
+PREFIX xsd:   <http://www.w3.org/2001/XMLSchema#>
+PREFIX dct:   <http://purl.org/dc/terms/>
+
+INSERT DATA {
+  GRAPH <http://mu.semte.ch/graphs/public> {
+
+      # Insert bestuurseenheid
+      <http://data.lblod.info/id/bestuurseenheden/66DB132B6ABCA6E6BF2DBBED> rdf:type	<http://data.vlaanderen.be/ns/besluit#Bestuurseenheid>, ;
+        <http://mu.semte.ch/vocabularies/core/uuid>	"66DB132B6ABCA6E6BF2DBBED" ;
+        skos:prefLabel	"Sint-Anna" ;
```

Comment:
Welzijnsvereniging is the bestuurseenheid classificatie. Classificaties get concatenated in many instances with the name (e.g. mock login list, toezicht bestuurseenheid concept schemes). I havent tested it yet but most likely this will result in: `Welzijnsvereniging Welzijnsvereniging Sint-Anna`

Reply:
Yes, you're right. Better to keep it as `Sint-Anna` then.